src/frontend/src/features/layouts/components/mailbox-panel/components/mailbox-actions/_index.scss

```
@@ -1,49 +1,28 @@
+@keyframes mailbox-panel-actions__spin {
```

Stylelint (17.10.0)

[error] Expected keyframe name "mailbox-panel-actions__spin" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)